rfcs/034-location_location_location/README.md

```diff
@@ -0,0 +1,218 @@
+  },
+  "url": "https://iiif.wellcomecollection.org/image/A0000681.jpg/info.json",
+  "credit": "Royal Veterinary College",
+  "license": {
+    "id": "cc-by-nc",
+    "label": "Attribution-NonCommercial 4.0 International (CC BY-NC 4.0)",
+    "url": "https://creativecommons.org/licenses/by-nc/4.0/",
+    "type": "License"
+  },
+  "accessConditions": [],
+  "type": "DigitalLocation"
+}
+```
+
+A digitised item (a IIIF Presentation location):
+
+```json
+{
+  "locationType": {
+    "id": "iiif-presentation",
+    "label": "IIIF Presentation API",
+    "type": "LocationType"
+  },
+  "url": "https://wellcomelibrary.org/iiif/b22398752/manifest",
+  "license": {
+    "id": "pdm",
+    "label": "Public Domain Mark",
+    "url": "https://creativecommons.org/share-your-work/public-domain/pdm/",
+    "type": "License"
+  },
+  "accessConditions": [
+    {
+      "status": {
+        "id": "open",
+        "label": "Open",
+        "type": "AccessStatus"
+      },
+      "type": "AccessCondition"
+    }
+  ],
+  "type": "DigitalLocation"
+}
+```
+
+</details>
+
+
+
+## What should a location tell you?
+
+When somebody looks at an item, here's what they want to know about location:
+
+*   **Am I allowed to see it?**
+    This is encoded by the *access conditions* field, which includes an *access status* (drawn from a fixed vocabulary: open, closed, requires-permission, etc.) and an *access description* (free-text, written by cataloguers).
+
+*   **If I'm allowed to see it, where can I find it?**
+    For example: where it sits on the physical shelves within the library, or a particular URL, or a IIIF resource.
+
+*   **If I'm allowed to see it, can I use it?**
+    This is the *license* on the item.
+
+    Note: Our current model only supports a license on digital locations.
+    Is this correct, or do we need to support licenses on physical locations also?
+
+We need to support filtering and aggregating by location.
+For example, users should be able to find all the material that's openly available online (if they're working remotely), or all the material in the library (if they're in the building).
+
+
+
+## What locations do we need to support?
+
+These are the types of location our model needs to support:
+
+*   **Open shelves.**
+
+    This should include a *shelf location* (e.g. History of Medicine Collection) and a *shelfmark* (e.g. TW.AL) so that a user can find it.
+
+*   **Closed stores.**
+
+    Although the MARC records include a shelf location and a shelfmark, the Collections team would prefer this not to be exposed publicly.
+
+    Do we need to expose this information for the requesting service?
+
+    -   We don't need to expose the full detail of our closed stores, but do we want to expose the difference between onsite/offsite?
+        The latter will take longer to retrieve, and may require a manual request to Library Enquiries.
+        Is this something that should be exposed by the Catalogue API, or only by the Requesting service?
```

Comment:
Where material is not held onsite, this needs to be clear on catalogue records (in order to manage user expectations) but I'm not sure whether that means in the catalogue API. 

Related, IMHO it would be good to work towards the eradication of manual requests so that we can provide (as Accreditation requires) a proper audit trail of all requests.

Reply:
As above, I'm not going to expose this distinction in the external catalogue API right now. We can revisit it when we build out requesting.